server/filestore.go

```diff
@@ -791,7 +800,6 @@ func (mb *msgBlock) rebuildState() (*LostStreamData, error) {
 
 			mb.msgs++
 			mb.bytes += uint64(rl)
-			mb.rbytes += uint64(rl)
```

Comment:
This was essentially done by line 670?
mb.rbytes = uint64(len(buf))

Reply:
Yes that was a bug, could not figure out why my compaction and rebuild state was saying same for rbytes ;)